superset/assets/javascripts/explorev2/stores/controls.jsx

```diff
@@ -550,7 +550,8 @@ export const controls = {
     type: 'SelectControl',
     freeForm: true,
     label: 'Number format',
-    default: D3_TIME_FORMAT_OPTIONS[0],
+    renderTrigger: true,
```

Comment:
if use 'D3_TIME_FORMAT_OPTIONS[0]', Number format field is blank in the UI somehow.